packages/flutter/lib/src/material/refresh_indicator.dart

```diff
@@ -174,7 +187,33 @@ class RefreshIndicator extends StatefulWidget {
     this.semanticsValue,
     this.strokeWidth = RefreshProgressIndicator.defaultStrokeWidth,
     this.triggerMode = RefreshIndicatorTriggerMode.onEdge,
-  }) : _indicatorType = _IndicatorType.adaptive;
+  })  : _indicatorType = _IndicatorType.adaptive,
+        onStatusChange = null;
+
+  /// {@tool dartpad}
```

Comment:
The doc comments inside `{@tool dartpad}...{@end-tool}` belong to the example, not the actual constructor. The constructor needs its own descriptive API docs.